src/domain/proposal.ts

```diff
@@ -186,12 +208,38 @@ export function updateGSProposal(
   createdAt: BigInt,
   avatarAddress: Address,
   descriptionHash: string,
+  schemeAddress: Address,
 ): void {
   let proposal = getProposal(proposalId.toHex());
   proposal.dao = avatarAddress.toHex();
   proposal.genericScheme = proposalId.toHex();
   proposal.createdAt = createdAt;
   proposal.descriptionHash = descriptionHash;
+  proposal.scheme = crypto.keccak256(concat(avatarAddress, schemeAddress)).toHex();
+  if (proposal.organizationId != null && proposal.scheme != null) {
+    setScheme(proposal.organizationId, proposal.scheme);
```

Comment:
why setting scheme at gpque on each proposal ?

Reply:
We can add a check if that's already set...

Reply:
not sure it is the right place here.